chapter_convolutional-modern/googlenet.md

```diff
@@ -392,7 +395,11 @@ def b5(self):
     if tab.selected('jax'):
         return nn.Sequential([Inception(256, (160, 320), (32, 128), 128),
                               Inception(384, (192, 384), (48, 128), 128),
-                              lambda x: nn.avg_pool(x, (1, 1)),
+                              # Flax doesn't provide a GlobalAvgPool2D layer
+                              lambda x: nn.avg_pool(x,
+                                                    window_shape=x.shape[1:3],
+                                                    strides=x.shape[1:3],
+                                                    padding="VALID"),
```

Comment:
Use single quotes consistently.
Besides, can we also apply consistency to using either lower-case or upper-case strings for `padding`? Currently it's mixed: 'same', 'VALID'

Reply:
Fixed!